### ISSUE TITLE
plugin dependency issue on androidx.emoji2:emoji2-views-helper:1.4.0 capacitor 5

### ISSUE THREAD

**Issue by Aksoom-Hussain** [capacitor-community/stripe#331]
**Platform**

**Android**

unable to run app or get the app build successfully after updating from 5.0.0 to 5.0.6
need seperate version mention for v5 as v6 is still in beta and other plugin also to need migration 

on running app in android 14
the error we get

```
 java.lang.SecurityException: com.vajjan.partner: One of RECEIVER_EXPORTED or RECEIVER_NOT_EXPORTED should be specified when a receiver isn't being registered exclusively for system broadcasts
```
[https://github.com/ionic-team/capacitor-plugins/issues/1994](url)

**Comment by inorganik** [comment]
There is some more detail here: https://github.com/ionic-team/capacitor-plugins/issues/1994

The issue can be fixed by updating the target SDK to 34, but this change should only roll out with capacitor v6 and currently this plugin is (should be) compatible with capacitor 5, which targets 33.

**Comment by rdlabo** [comment]
Thanks for the issue, seems to be caused by the stripeAndroidVersion.

https://github.com/capacitor-community/stripe/commit/d31584c23f968d0184882165b116d0a2e5ab644a#diff-b599c22598368631cc43e7e02acf0a 52f84a6c7d0dd7e212414c822f2dbc6821

Can you try setting the stripeAndroidVersion to `20.32.+` or lower in the project `variables.gradle` to see if that fixes it?

**Comment by Aksoom-Hussain** [comment]
@rdlabo thank you for your reply, 
i fix it by downgrading the package, although i feel the documentation need to be mention  fix by separating capacitor v5 and v6 version as v5 support android version 33

**Comment by rdlabo** [comment]
Thanks, v6 support is in progress at https://github.com/capacitor-community/stripe/pull/332 and does not need to be considered here.

We have released 5.4.3-1 to see if this is caused by an issue with the Stripe Android version. Please try this version.

```
% npm install @capacitor-community/stripe@5.4.3-1
% npx cap update
```

**Comment by inorganik** [comment]
@rdlabo Unfortunately installing @capacitor-community/stripe@5.4.3-1 did not work - did you test it first?
![Screenshot 2024-01-30 at 10 35 11 AM](https://github.com/capacitor-community/stripe/assets/109553/e788a27a-0ca8-4879-896f-8be2e1f57132)

I get a successful build if I roll back to v5.2.0

**Comment by Aksoom-Hussain** [comment]
> @Aksoom-Hussain Thanks, v6 support is in progress at #332 and does not need to be considered here.
> 
> We have released 5.4.3-1 to see if this is caused by an issue with the Stripe Android version. Please try this version.
> 
> ```
> % npm install @capacitor-community/stripe@5.4.3-1
> % npx cap update
> ```

have tried your patch the issue still exist
